src/clj/rems/db/applications/legacy.clj

```diff
@@ -122,7 +122,7 @@
            :value
            (cond
              (nil? application-id) ""
-             (= "attachment" (:type field)) (or (:filename (db/get-attachment query-params)) "")
+             ;;(= "attachment" (:type field)) (assert false "attachments not supported")
```

Comment:
need to figure out what to do about this case

Reply:
It's used only in application conversion and in test data. Since there is no migration for attachments, it should be OK to let the conversion crash if there are attachments.